src/js/player.js

```diff
@@ -2763,6 +2763,9 @@ class Player extends Component {
    * and calls `reset` on the tech`.
    */
   reset() {
+    if (this.tech_) {
+      this.tech_.clearTracks(['text']);
```

Comment:
Nitpicking, but this could save a couple of characters since `clearTracks()` puts a string argument into an array.

https://github.com/videojs/video.js/blob/237041698c38177b99de5995b13c298c837ed4a9/src/js/tech/tech.js#L391-L392